content/en/developers/integrations/_index.md

```diff
@@ -106,8 +106,12 @@ API-based integrations can submit telemetry—such as metrics, traces, logs, and
 
 Technology Partners write and host the implementation code that makes up the integration. [Creating an API integration][1] works well for Technology Partners building a connector between Datadog and another SaaS platform.
 
-
-
+## Requirements
+All integrations must include the following:
+* An Out-of-the-box Integration Dashboard
+* 3 or more images for your tile
+* OAuth - for API Integrations only
+* A log pipeline - If your integration sends logs to Datadog
```

Comment:
Should we link out to the associated docs or would this be too much?

Reply:
Since these are in the markdown file and won't be hyperlinks, im not sure they'd get too many clicks if they are buried in here. just my 2c

Reply:
Agreed - we will keep as is